src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssString = sourceString;
+  const cssObject = {};
+
+  cssString
+    .split(';')
+    .map((pair) => {
```

Comment:
method `map`  should return new array and don't mutate current, because map it a `clear function`
you can use method `forEach` instead of `map`
but in this case, it's better to use method reduced, because you can avoid creating additional variable `cssObject `